app/database/operator/server_data_operator/transformers/post.ts

```diff
@@ -170,10 +169,10 @@ export const transformSchedulePostsRecord = ({action, database, value}: Transfor
         scheduledPost.files = raw?.metadata?.files ?? emptyFileInfo;
         scheduledPost.metadata = raw?.metadata ?? null;
         if (raw.priority) {
-            if (!scheduledPost.metadata) {
```

Comment:
is scheduledPost.metadata never undefined/null?

Reply:
`scheduledPost.metadata` is always `{}` if no file info exists.